packages/pkgsrc.rb

```diff
@@ -0,0 +1,125 @@
+  end
+
+  def self.postinstall
+    puts
+    puts "You may wish to have the vulnerabilities file downloaded daily so that it"
+    puts "remains current. This may be done by adding an appropriate entry to the root"
+    puts "users crontab(5) entry. For example, the entry"
+    puts
+    puts "# Download vulnerabilities file"
+    puts "0 3 * * * #{CREW_PREFIX}/pkg/sbin/pkg_admin fetch-pkg-vulnerabilities >/dev/null 2>&1"
+    puts "# Audit the installed packages and email results to root"
+    puts "9 3 * * * #{CREW_PREFIX}/pkg/sbin/pkg_admin audit |mail -s 'Installed package audit result' \\"
+    puts "            root >/dev/null 2>&1"
+    puts
+    puts "will update the vulnerability list every day at 3AM, followed by an audit at"
+    puts "3:09AM. The result of the audit are then emailed to root. On NetBSD this may be"
+    puts "accomplished instead by adding the following line to /etc/daily.conf:"
+    puts
+    puts "fetch_pkg_vulnerabilities=YES"
+    puts
+    puts "to fetch the vulnerability list from the daily security script. The system is"
+    puts "set to audit the packages by default but can be set explicitly, if desired (not"
+    puts "required), by adding the following line to /etc/security.conf:"
+    puts
+    puts "check_pkg_vulnerabilities=YES"
+    puts
+    puts "Both pkg_admin subcommands can be run as as an unprivileged user,"
+    puts "as long as the user chosen has permission to read the pkgdb and to write"
+    puts "the pkg-vulnerabilities to #{CREW_PREFIX}/pkg/pkgdb."
+    puts
+    puts "The behavior of pkg_admin and pkg_add can be customised with"
+    puts "pkg_install.conf.  Please see pkg_install.conf(5) for details."
+    puts
+    puts "If you want to use GPG signature verification you will need to install"
+    puts "GnuPG and set the path for GPG appropriately in your pkg_install.conf."
+    puts "==========================================================================="
+    puts "===> running: #{CREW_PREFIX}/bin/install-sh -d -o chronos -g lp #{CREW_PREFIX}/pkg/etc"
+    puts
+    puts "==========================================================================="
+    puts
+    puts "An example mk.conf file with the settings you provided to 'bootstrap'"
+    puts "has been created for you. It can be found in:"
+    puts
+    puts "      #{CREW_PREFIX}/pkg/etc/mk.conf"
+    puts
+    puts "You can find extensive documentation of the NetBSD Packages Collection"
+    puts "in #{CREW_PREFIX}/doc/pkgsrc.txt."
+    puts
+    puts "Thank you for using pkgsrc!".lightgreen
+    puts
+    puts "To finish the installation, execute the following:".lightblue
+    puts "echo 'export PATH=\"#{CREW_PREFIX}/pkg/bin:#{CREW_PREFIX}/pkg/sbin:\$PATH\"' >> ~/.bashrc".lightblue
+    puts "echo 'export MANPATH=\"#{CREW_PREFIX}/pkg/man:\$MANPATH\"' >> ~/.bashrc".lightblue
+    puts "source ~/.bashrc".lightblue
+    puts
+    puts "To install packages, execute the following:".lightblue
```

Comment:
We should probably add that packages can also be installed using binaries.

`pkg_add ftp://ftp.NetBSD.org/pub/pkgsrc/packages/Net-BSD/i386/3.1_2007Q2/shells/bash-3.2.9.tgz`

Reply:
We could but this package already has all the source to build and install.  I suppose if you wanted a more recent version than the quarterly release, this would be beneficial.  I was thinking about adding the pkgin command in the future to download and install binary packages precompiled for Chrome OS.  I'd like to contact Benny Siegert to ask him if it could be added to the pkgsrc repo at `ftp://ftp.NetBSD.org/pub/pkgsrc/packages/Linux/<arch>/ChromeOS_2020Q2/All` or something like that.